docs/index.md

```diff
@@ -45,47 +44,18 @@ Despite this broad scope, our *short-term definition of success* requires that w
+## User Experience and Data Flow
 
-### Connecting Health Wallet to Issuer (optional)
-* Establish a SMART on FHIR authorization with an Issuer including read access to any resources that will be present in Health Cards (e.g., Patient, Immunization, Observation, DiagnosticReport).
-
-### Getting credentials into Health Wallet
-* Required method: File download
-* Required method: Print QR on paper card, or scan QR into software
-* Optional method: [FHIR API Access](#healthwalletissuevc-operation)
-
-### Presenting credentials to Verifier
-* Optional method: QR presentation
-* Optional method: On-device SDKs (e.g., for verifier-to-holder app-to-app communications)
+* **User Receives** a Health Card from an Issuer. The Health Card is a signed data artifact that the user can obtain through any of these methods:
+  * issuer offers a Health Card on paper or PDF, including a QR code (required method)
+  * issuer offers a Health Card for download as a `.smart-health-card` file (required method)
+  * issuer hosts a Health Card for [FHIR API access](#healthwalletissuevc-operation) via a compatible Health Wallet application. This workflow includes a SMART on FHIR authorization step with an Issuer, where the user grants  read access to any resources that will be present in Health Cards (e.g., `Patient`, `Immunization`, `Observation`, `DiagnosticReport`)
+* **User Saves** a Health Card, whether on paper or digitally.
+* **User Presents** a Health Card to a Verifier. Presentation includes explicit user opt-in and approval, and may involve displaying a QR code, sharing a file, or using an on-device SDK (e.g., for verifier-to-holder app-to-app communications)
 
 ## Trust
 
-Which issuers can participate, which test results should be considered, and how do verifiers learn this information?
-
-At a _pilot project level_:
-
-### Which Issuers can participate?
-* We'll work with a willing set of issuers and define expectations/requirements
-* Verifiers will learn the list of participating issuers out of band; each issuer will be associated with a public URL
-* Verifiers will discover public keys associated with an issuer via `/.well-known/jwks.json` URLs
-* For transparency, we'll publish a list of participating organizations in a public directory
-* In a _post-pilot deployment_, a network of participants would define and agree to a formal Trust Framework
+Which issuers should be trusted to issue Health Cards, and how do verifiers learn this information? Ultimately any verifier can make its own decisions, but a "trust framework" is a useful way to externalize this work and drive toward more consistent practices. The SMART Health Cards IG is designed to operate independent of any trust framework, while allowing trust frameworks to be layered on top. We anticipate trust frameworks will emerge to meet different jurisdictional and use case driven requirements. In all cases, verifiers can discover public keys associated with an issuer via `/.well-known/jwks.json` URLs.
```

Comment:
Maybe:
> Health Cards from which issuers should be trusted, and how do verifiers [...]
?

Reply:
Thanks -- I've taken another pass at this text.